instrumentation/netty/netty-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/NettyClientSingletons.java

```diff
@@ -3,10 +3,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.instrumentation.netty.v4_1.client;
+package io.opentelemetry.javaagent.instrumentation.netty.v4_1;
```

Comment:
just checking that package was intentional?

Reply:
It was intentional because after the split this was the last class left in the package.

Reply:
oh makes sense 👍